app/features/login/login-form.tsx

```diff
@@ -0,0 +1,110 @@
+import { useOpenSecret } from '@opensecret/react';
+import { Link } from '@remix-run/react';
+import { useEffect } from 'react';
```

Comment:
we should add "rules": {"correctness":  "noUnusedImports": "error"} biome.jsonc

Reply:
added